gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -122,8 +122,7 @@ public static void submitJobToQueue(
   static void waitJobInitialization(
       HelixManager helixManager,
       String workFlowName,
-      String jobName,
-      long timeoutMillis) throws Exception {
```

Comment:
why removing?

Reply:
time for job initialization was not configurable earlier also. it was being called only from one palce with fix value "300_000" . "300_000" also had a typo, i removed it.